src/sentry/api/serializers/models/notification_setting.py

```diff
@@ -29,7 +29,7 @@ def get_attrs(
         """
         from sentry.models import NotificationSetting
 
-        actor_mapping = {target.actor_id: target for target in item_list}
```

Comment:
`item_list` is a list of `NotificationSetting` objects. The actor mapping needs to map `actor_id` to `NotificationSetting`.